src/main/kotlin/racing/domain/Car.kt

```diff
@@ -12,20 +17,5 @@ class Car(
 
     fun samePosition(position: Int): Boolean = this.position == position
 
-    fun getPosition(): Int = this.position
-
-    override fun equals(other: Any?): Boolean {
-        if (this === other) return true
-        if (javaClass != other?.javaClass) return false
-
-        other as Car
-
-        if (position != other.position) return false
-
-        return true
-    }
-
-    override fun hashCode(): Int {
-        return position
-    }
+    fun name() = name.value
```

Comment:
[Kotlin Coding Convention](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names)에 따르면 method 네이밍은 `동사구`를 사용하라고 가이드하고있어요. 😃

아래와 같이 변경해보면 어떨까요? 😃

```suggestion
    val rawName: String
        get() = name.value
```